indexer/config/devnet.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type ethereumContracts struct {
```

Comment:
does parsing fail if you leave out the unneeded fields? If we simply supply `json` tags to the `config.L1Contracts` structs can we deserialize directly into that?

Reply:
Hmm curious too. It's a common.Address not a string so my guess is no but curius if it works just from setting type to common.Address

Reply:
My guess is yes we can leave out the unneeded fields but will give that a try to

Reply:
Just checked and `common.Address` implements the `UnmarshalJSON` interface so it should work!

Reply:
nice TIL about UnmarshalJSON